bin/generate_dist

```diff
@@ -4,76 +4,63 @@
 require_relative "../config/environment"
 require_relative "../config/cli"
 
-class GenerateDist
-  class << self
-    def call(all_locales: false)
-      @cli = CLI.new([:version]) do |opts|
-        opts.on("-V VERSION", "--version=VERSION", "Version of the documentation")
-      end
-      @cli.parse!(ARGV)
-      @cli.options.version ||= @cli.read_file("VERSION").strip
-
-      puts("Generating `#{@cli.options.version}` distribution files...")
-      @cli.options_status
-
-      return generate_dist_files("en") unless all_locales
-
-      locales = Dir.glob("data/localizations/categories/*.yml").map { |f| File.basename(f, ".yml") }
+@cli = CLI.new([:version, :locales]) do |opts|
+  opts.on("-V VERSION", "--version=VERSION", "Version of the documentation")
+  opts.on("-l LOCALES", "--locales=LOCALES", Array, "Locales to generate")
+end
+@cli.parse!(ARGV)
+@cli.options.version ||= @cli.read_file("VERSION").strip
 
-      locales.each do |locale|
-        generate_dist_files(locale)
-      end
-    end
+@cli.options.locales ||= ["en"]
+if @cli.options.locales.include?("ALL")
```

Comment:
small aside: I'm unsure if OptionParser downcases for us or not (just something to vet locally)